dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo

```diff
@@ -44,13 +44,11 @@ model PIAntiWindUpTable "Proportional Integrator with anti-windup and table-base
   parameter Real Y0 "Start value of output";
 
 equation
-  when time >= 0 and u.value <> pre(u.value) then
+  if time >= 0 then
     y.value = combiTable1D.y[1];
-  elsewhen time >= 0 and u.value == pre(u.value) then
+  else
     y.value = pre(y.value);
-  elsewhen time < 0 then
-    y.value = 0;
-  end when;
+  end if;
   connect(add1.y, integrator.u) annotation(
```

Comment:
According to the user's guide : One blank line between equations and a when equation or an if equation

Please add a blank line after end if;

Reply:
done